lms/djangoapps/teams/management/commands/tests/test_reindex_course_team.py

```diff
@@ -0,0 +1,98 @@
+""" Tests for course_team reindex command """
+import sys
+import contextlib
+import ddt
+import mock
+
+from mock import patch
+from django.core.management import call_command, CommandError
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
+from opaque_keys.edx.keys import CourseKey
+from teams.tests.factories import CourseTeamFactory
+from teams.search_indexes import CourseTeamIndexer
+from search.search_engine_base import SearchEngine
+
+
+@contextlib.contextmanager
+def nostderr():
```

Comment:
Could this go in the new common/test/utils.py class that @peter-fogg is introducing? I see it is already in at least one other place in our codebase.